README.md

```diff
@@ -1 +1,172 @@
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+  - [👥 Author](#-author)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+  
+<!-- PROJECT DESCRIPTION -->
+
+# 📐 [Vet Clinic:] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+Vet Clinic Database is relational database creates initial data structure for a vet clinic. It also has a table that store animals' information, insert some data into it, and query it.
+
+
+
+**[Vet Clinic:: setup]** It is Vet Clinic using PostgreSQL.
+
+## 🛠 Built With <a name="built-with">
+
+PostgreSQL
+
+</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Gitflow
+- Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Under construction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [ ]  install PostgreSQL
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+- Open the command prompt/terminal and navigate to the created directory.
+
+- On the terminal run this command git clone `https://github.com/AbdifatahYasin1/Vet-clinic.git`
+
+- Go to the repository folder using command prompt
+
+
+### Install
+
+Install this project with:
+
+Cloning steps:
+
+1. - [ ] Open on the `Terminal`
+2. - [ ] Run `https://github.com/AbdifatahYasin1/Vet-clinic.git`
+3. - [ ] cd Vet-clinic
+4. - [ ] Open in the IDE
+5. - [ ] open with the liver server
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+## Author
+
+👤 **Abdifatah**
+
+- GitHub: [@AbdifatahYasin1](https://github.com/AbdifatahYasin1)
+- Twitter: [@CabdifataaxYuusuf](https://twitter.com/CabdifataaxYy)
+- LinkedIn: [Cabdifataax Yaasiin](https://www.linkedin.com/in/cabdifataax-yaasiin-69977019a/)
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
```

Comment:
- [ ] Kindly fill out this show support section of your README as it is required